designer/server/src/common/nunjucks/context/index.js

```diff
@@ -3,12 +3,13 @@ import { basename, join } from 'node:path'
 
 import { SCOPE_READ } from '~/src/common/constants/scopes.js'
 import { getUserSession } from '~/src/common/helpers/auth/get-user-session.js'
+import environmentTagHelper from '~/src/common/helpers/environment-tag-helper.js'
 import { createLogger } from '~/src/common/helpers/logging/logger.js'
 import { buildNavigation } from '~/src/common/nunjucks/context/build-navigation.js'
 import config from '~/src/config.js'
 
 const logger = createLogger()
-const { phase, serviceName } = config
+const { phase, serviceName, env, isTest, isProduction } = config
```

Comment:
`config.env` is only `development`/`test` locally. In a live environment it always runs as `production`.